scripts/build_index.rb

```diff
@@ -20,6 +20,6 @@
     end
 end
 
-commands = commands.map do |k,v| v end
+commands = commands.sort.map do |k,v| v end
```

Comment:
My Ruby's not great.... but how does it sort the commands since each object has `{name, platform}`? Should we specify something like `sort(:name)`?